js/src/data/test/reducer.test.js

```diff
@@ -38,6 +38,7 @@ describe( 'reducer', () => {
 				contact: null,
 			},
 			ads_campaigns: null,
+			all_ads_campaigns: null,
```

Comment:
💅 / ❓  

Not sure if this is temporary until finishing the migration but if this is going to be there for a long time maybe we could consider to group the `all` and `available` campaigns  inside `ad_campaigns` instead of creating a separated state.

Reply:
Yeah, we could consider that option in the future, we will need to see which parts of the code would need to be updated and what other side effects could have. I can have a look into that in a different PR.